Lab3.ipynb

```diff
@@ -0,0 +1,405 @@
+      "text/plain": [
+       "<PIL.JpegImagePlugin.JpegImageFile image mode=RGB size=302x75 at 0x7F40B5098AD0>"
+      ]
+     },
+     "execution_count": 115,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# 1.A.3:\n",
+    "Image.open('Lab3.1.jpg')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Problem 1.A.3: Using the equation above, we can take the integral from -infinity to -x and the integral from x to infintiy and set them equal to the probability of 5 sigma; since that's what we deem significant enough. This will yield what values of temperature are significant enough to be considered not created by the background. In order to find x we can utilize the equation (x-u)/(sigma). Therefore any temperature not in the range of 10-14 we can consider stastically significant enough to remove as bad data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 116,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Sigma value: 5\n",
+      "Pos True: 100000\n",
+      "Pos False: 2\n",
+      "Neg True: 6\n",
+      "Neg False: 0\n"
+     ]
+    }
+   ],
+   "source": [
+    "# 1.A.4: \n",
+    "sigma=5\n",
+    "goodKept = goodData[np.where((goodData< 12+sigma*.4) & (goodData> 12-sigma*.4))]\n",
+    "goodDiscard = len(goodData) - len(goodKept)\n",
+    "badData = np.asarray(badData)\n",
+    "badKept = badData[np.where((badData>12-sigma*.4) & (badData<12+sigma*.4))]\n",
+    "badDiscard = len(badData) - len(badKept)\n",
+    "print('Sigma value:', sigma)\n",
+    "print('Pos True:', len(goodKept))\n",
+    "print('Pos False:', len(badKept))\n",
+    "print('Neg True:', badDiscard)\n",
+    "print('Neg False:', goodDiscard)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Problem 1.A.4: According to my truth table for a 5 sigma stastistical significance probability, we are keeping 2 bad data values and all the good data values. "
```

Comment:
Problem 1.A.4:
- output makes sense. You could put it in a pandas table (since you imported pandas package) but otherwise looks good. If you don't end up using pandas, you can remove the import statement in your first notebook cell
- typo: stastistical --> statistical; the word "probability" could be removed from your sentence and just say statistical significance